test/user_test.jl

```diff
@@ -0,0 +1,28 @@
+include("../src/SMTPClient.jl")
+#= import Base64: Base64EncodePipe
+import Dates
+include("../src/mime_types.jl")
+include("../src/user.jl") =#
+using .SMTPClient
+
+opt = SendOptions(
+  isSSL = true,
+  username = "rmsrosauk@gmail.com",
+  passwd = "adgjta1212"
+)
+
+url = "smtps://smtp.gmail.com:465"
+from = "<rmsrosauk@gmail.com>"
+
+message = "Here is a version for the PR, with attachment"
+to = ["<rrosa@im.ufrj.br>", "<rmsrosa@gmail.com>"]
```

Comment:
these look like real email addresses. Are you happy to have them present in the tests?

Reply:
Ops, certainly not. I was not supposed to have committed these files, but I guess I did, by mistake. I should get rid of them and change the pwd cause I think it will stay in the commit history.

Reply:
yes, please change your password. I'll squash and merge, so they won't be on master, but that's still not fully safe.

Reply:
ok, I just changed, thanks for the warning!

Reply:
and I spoke in "plural" form, but that is actually the only file that shouldn't have been committed.